jsonnet/prometheus.libsonnet

```diff
@@ -163,6 +163,17 @@ function(params)
       ],
     },
 
+    roleConfig+: {
```

Comment:
to be consistent with Prometheus UWM and Thanos ruler, it might be better to crete an `alertManagerRoleBinding` entry instead of extending that role. It would also avoid duplicating the rule.